### ISSUE TITLE
Fedora 31 release

### ISSUE THREAD

**Issue by teg** [osbuild/osbuild-composer#67]
Write a spec file and get osbuild-composer into fedora 31. The spec file should be accepted into fedora, work with packit and work with the (possibly adjusted) make-repo.sh tool in the tools directory. Once this is finished, and a first release out (passing minimal manual testing), immediately start the process of inclusion into RHEL8.2.

**Comment by msehnout** [comment]
I don't think this qualifies as a one-cycle issue. The package review process itself is a 2 weeks task because it is blocked on the reviewer and we cannot anticipate his/her response time.

**Comment by ondrejbudai** [comment]
Yeah, but I think we should have it scheduled in the cycle to visibly allocate the time required to manage the process. I am OK with moving it to the next cycle, it's an odd one.

**Comment by msehnout** [comment]
Sure, I'd also like to see the spec file merged, but I think this issue needs to be split in order to fit into a single cycle.

**Comment by ondrejbudai** [comment]
The first version of specfile is already merged, currently we need to do two things:

- fix the issue with make-repo.sh (see #53)
- start the review process (blocked by first release)

**Comment by teg** [comment]
Good point about possibly being blocked. I think we should still try to get this done in a cycle, but if we end up blocked on external people so be it (and we'll just have to let it go over several cycles).

This issue is a bit special as the work is very async (blocked on other things), so hopefully it will be easy to do in parallel with other things.

**Comment by teg** [comment]
I now tagged the first release. As to the make-repo.sh issue, will this be made easier by the specfile targeting f31, rather than f30?

**Comment by ondrejbudai** [comment]
I don't think it matters. But as I said I need to investigate it more.

**Comment by teg** [comment]
This is close to being done, see: https://bugzilla.redhat.com/show_bug.cgi?id=1768774

**Comment by ondrejbudai** [comment]
I've just sent osbuild-composer to f31 updates-testing: https://bodhi.fedoraproject.org/updates/FEDORA-2019-5ba07c030a

@teg packit is still missing, but I think we should create a separate issue for this task and schedule it for later cycles. If you don't think so, feel free to reopen this issue. :)
